tsp-output/@azure-tools/typespec-csharp/app/DemoApp/Program.cs

```diff
@@ -0,0 +1,48 @@
+﻿using OpenAI;
+using OpenAI.Models;
+using System.ClientModel;
+
+using AzureOpenAI.Models;
```

Comment:
Once you add the namespace with the extension methods/properties, you can use an unbranded model like an Azure model.  How do we make sure people instantiate the client as an Azure client and only send Azure properties to the Azure service?